contracts/examples/DXswapRelayer.sol

```diff
@@ -0,0 +1,325 @@
+pragma solidity =0.6.6;
+
+import './OracleCreator.sol';
+import './../interfaces/IDXswapFactory.sol';
+import './../interfaces/IDXswapRouter.sol';
+import './../libraries/TransferHelper.sol';
+import './../interfaces/IERC20.sol';
+import './../libraries/SafeMath.sol';
+
+contract DXswapRelayer {
+    using FixedPoint for *;
+    using SafeMath for uint256;
+
+    event Pool(
+        address _tokenA,
+        address _tokenB,
+        uint256 _amountTokenA,
+        uint256 _amountTokenB,
+        uint256 _minA,
+        uint256 _minB,
+        uint256 _liquidity
+    );
+
+    struct Commitment {
```

Comment:
I see these more "Orders" than "Commitments" DXdao creating orders to add/remove/swap tokens into DXswap :grimacing:

Reply:
@AugustoL true, thanks for catching this!